src/main/java/nl/thehyve/ocdu/soap/ResponseHandlers/ImportDataResponseHandler.java

```diff
@@ -20,14 +20,24 @@
     /**
      * Checks if an error occurred on the OpenClinica-side and reports it back as the
      * return value
-     *
      * @param response the SOAP-response.
      * @return a non <code>null</code> error code.message if an error occurred. Some are reported by the OpenClinica-WS
      * instance at url. Returns <code>null</code> if everything went OK.
      * @throws Exception if a technical error occurs.
      */
     public static String parseImportDataResponse(SOAPMessage response) throws Exception {
-        return parseGenericResponse(response, ".//importDataResponse");
```

Comment:
hmm... this is not good. Because of rebasing on the last committ of the master (at the time) my commit modyfing this class is treated as older one and is overriden.